app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,86 @@
+class TasksController < ApplicationController
+  def index
+    @tasks = Task.all #on Task class #AR makes this available on this model and all the records of the db
+    @header = "Fresh Prince To-Do List"
+    @footer = "< Create New Task >"
+    @path = "tasks/new"
+    @footer2 = "< See the Homies >"
+    @path2 = "/people"
+    render :index
+  end
+
+  def new
+    @new_task = Task.new
+    @people = Person.all.map { |person| person.name }
+    @header = "Add a Task"
+    @footer = "< Back to All the Tasks >"
+    @path = "/"
+  end
+
+  def create
+    @new_task = Task.new(create_params[:task]) #pass params to diff function which should return params.require
+    @new_task.save
+    redirect_to "/" #redirect to / which will call index method and render tasks. render triggers that erb and redirects go somewhere else and triggers erb there
+  end
+
+  def edit
+    @task = Task.find(params[:id])
+    @header = "Edit Your Task, Man"
+    @footer = "< Back to All the Tasks >"
```

Comment:
I like the way that you used these variables to allow for dynamic header and footer values.